physical/raft/logstore/bolt_store.go

```diff
@@ -105,8 +114,40 @@ func (b *BoltStore) initialize() error {
 	return tx.Commit()
 }
 
+func (b *BoltStore) emitMetrics() {
+	for {
+		select {
+		case <-b.doneCh:
+			return
+		default:
+			stats := b.conn.Stats()
+			txstats := stats.TxStats
+			metrics.SetGauge([]string{"raft", "bolt_store", "freelist", "free_pages"}, float32(stats.FreePageN))
+			metrics.SetGauge([]string{"raft", "bolt_store", "freelist", "pending_pages"}, float32(stats.PendingPageN))
+			metrics.SetGauge([]string{"raft", "bolt_store", "freelist", "allocated_bytes"}, float32(stats.FreeAlloc))
+			metrics.SetGauge([]string{"raft", "bolt_store", "freelist", "used_bytes"}, float32(stats.FreelistInuse))
+			metrics.SetGauge([]string{"raft", "bolt_store", "transaction", "started_read_transactions"}, float32(stats.TxN))
+			metrics.SetGauge([]string{"raft", "bolt_store", "transaction", "currently_open_read_transactions"}, float32(stats.OpenTxN))
+			metrics.SetGauge([]string{"raft", "bolt_store", "page", "count"}, float32(txstats.PageCount))
+			metrics.SetGauge([]string{"raft", "bolt_store", "page", "bytes_allocated"}, float32(txstats.PageAlloc))
+			metrics.SetGauge([]string{"raft", "bolt_store", "cursor", "created"}, float32(txstats.CursorCount))
+			metrics.SetGauge([]string{"raft", "bolt_store", "node", "allocations"}, float32(txstats.NodeCount))
```

Comment:
Same here: `count` instead of `allocations`?

Reply:
Sure